full.rollbar/src/full/rollbar/middleware.clj

```diff
@@ -0,0 +1,27 @@
+(ns full.rollbar.middleware
+  (:require [clojure.core.async :refer [<!]]
+            [full.http.client :refer [req>]]
+            [full.async :refer [go-try]]
+            [full.rollbar.core :as rollbar]
+            [full.core.config :refer [defoptconfig]]
+            [camelsnake.core :refer [->snake_case]]))
+
+(defoptconfig rollbar-access-token :rollbar :access-token)
+(defoptconfig rollbar-environment :rollbar :environment)
```

Comment:
I think this is a fair point.  I wanted core to not have config, but it's in middleware anyways, so... I'm in agreement.